src/quicktype-core/language/Python.ts

```diff
@@ -408,6 +410,16 @@ export class PythonRenderer extends ConvenienceRenderer {
         return this.typeHint(" -> ", this.withTyping(type));
     }
 
+    protected sortClassProperties(p: ClassProperty, jsonName: string): number | string {
+        if (this.getAlphabetizeProperties()) {
+            return jsonName;
+        }
+        if (this.pyOptions.features.dataClasses) {
+            return p.type instanceof UnionType && Boolean(nullableFromUnion(p.type)) ? 1 : 0;
```

Comment:
`nullableFromUnion(p.type) !== null` instead of `Boolean(nullableFromUnion(p.type))`, please.